src/routes/trip.route.js

```diff
@@ -10,11 +10,11 @@ import UserValidation from '../validations/user.validation';
 
 const router = express.Router();
 
-router.post('/one-way-trips', authMiddleware.checkUserLoggedIn, authMiddleware.checkIfUserHaveManager, oneWayTripValidation, TripController.requestOneWayTrip); // One way trip route
-router.post('/return-trip', authMiddleware.checkUserLoggedIn, authMiddleware.checkIfUserHaveManager, TripValidation, TripController.requestReturnTrip); // Return trip route
-router.get('/requests/:userId', authMiddleware.checkUserLoggedIn, requestValidation, TripController.userTripRequestList); // user request list route
-router.post('/multi-city-trips', authMiddleware.checkUserLoggedIn, authMiddleware.checkIfUserHaveManager, multiCityTripValidation, TripController.requestMultiCityTrip);
+router.post('/trips/one-way', authMiddleware.checkUserLoggedIn, authMiddleware.checkIfUserHaveManager, oneWayTripValidation, TripController.requestOneWayTrip); // One way trip route
```

Comment:
I would expect that you will take the initiative to remove the common route prefix and add it to `routes/index.js` as a route context.